index.bs

```diff
@@ -1219,7 +1219,7 @@ following Web IDL.
     dictionary CollectedClientData {
         required DOMString           challenge;
         required DOMString           origin;
-        required DOMString           hashAlg;
+        required DOMString           hashAlgorithm;
```

Comment:
IIUC, #326 and #351 say that the CollectedClientData and PublicKeyCredentialParameters fields should have consistent names. Here you've switched which one uses the short form, but they're still inconsistent. Maybe unify on "alg"?